### PR TITLE
Improve TTS volume level for Android Auto

### PR DESCRIPTION
**Pull request by tt-driver-apps**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -15,21 +15,25 @@ buildscript {
...
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 26)
-    buildToolsVersion safeExtGet('buildToolsVersion', '26.0.3')
+    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
 
     defaultConfig {
-        minSdkVersion safeExtGet('minSdkVersion', 16)
-        targetSdkVersion safeExtGet('targetSdkVersion', 26)
+        minSdkVersion safeExtGet('minSdkVersion', 23)
```
Comment (casparvanderheiden-tomtom) [inline android/build.gradle]: Can we increase the minSdkVersion?

Comment (tt-driver-apps) [reply android/build.gradle]: I copied the version numbers from the react-native-voice repo, let me know which versions it should be instead and I'll change it

Comment (casparvanderheiden-tomtom) [reply android/build.gradle]: I'm wondering if we get incompatibility issues with AmiGO, where the minSdkVersion is 21. But that means the same is true for rnv...

File: android/src/main/java/net/no_mad/tts/TextToSpeechModule.java [hunk 2]
```diff
@@ -217,12 +304,32 @@ public void speak(String utterance, ReadableMap params, Promise promise) {
...
-        }
+        // queue TTS utterance using single thread executor so that audio focus can be requested early
+        // when requesting to speak an utterance instead of waiting for it to be realy started to avoid
+        // nested requests when an utterance is requested to be spoken while another is still playing
+        executor.execute(() -> {
+            if (enableTestCode) {
+                testCodeReset();
+            }
+
+            isTtsCompleted = false;
```
Comment (casparvanderheiden-tomtom) [inline android/src/main/java/net/no_mad/tts/TextToSpeechModule.java:315]: Is this thread safe?

**Review by casparvanderheiden-tomtom — APPROVED** [review]

File: android/src/main/java/net/no_mad/tts/TextToSpeechModule.java (hunk 2 above)
Comment (tt-driver-apps) [reply android/src/main/java/net/no_mad/tts/TextToSpeechModule.java:315]: I think so yes. It is running in a single thread executor and all the code that sets it back to true is on response of what is running inside the executor.

File: android/build.gradle (hunk 1 above)
Comment (tt-driver-apps) [reply android/build.gradle]: Shall l change it to 21 then to be sure?

Comment (casparvanderheiden-tomtom) [reply android/build.gradle]: If there is no specific reason to use 23, then I guess that is the best.

**Review by SanderGols-TomTom — APPROVED** [review]

File: android/build.gradle (hunk 1 above)
Comment (tt-driver-apps) [reply android/build.gradle]: ok I changed it to 21

**Pull request merged**